spec/functional/templating_spec.rb

```diff
@@ -30,4 +30,88 @@
       EMR
     )).to eq('<h1>Hello, Dave</h1>')
   end
+
+  it 'works in multiline literals' do
+    expect(convert(
```

Comment:
Actually, I'll make a separate PR to clean up this stuff once it's merged in. I wanted to have a 'cleanup' PR which addresses a few things in the codebase. I'll approve the changes - nice job Dave!! 💯 💯 💯